packages/editor/templates/Coin/contracts/Coin.sol

```diff
@@ -1,35 +1,56 @@
+// Specifies that the source code is for a version
+// of Solidity greater than 0.5.10
 pragma solidity >=0.5.0 <0.7.0;
 
+// A contract is a collection of functions and data (its state)
+// that resides at a specific address on the Ethereum blockchain.
 contract Coin {
-    // The keyword "public" makes variables
-    // accessible from other contracts
+    // The keyword "public" makes variables accessible from outside a contract
+    // and creates a function that other contracts or SDKs can call to access the value
+
+    // An address stores addresses of contracts, or a hash of the public half
```

Comment:
Perhaps simpler to say `An address stores addresses of contracts or external (user) accounts`

Reply:
I was wondering that, but then the statement is only half correct, but at this point, it's probably not something a beginner needs to know.

Reply:
Your call but yes, I think we should focus on the level of understanding a beginner needs to know. I'm comfortable with not being completely correct.